### PR TITLE
Mdx

### PR DESCRIPTION
**Pull request by sarahbethfederman**
I am not sure if I did the bolt thingies right. Couldn't figure out if there was a correct way to generate the package.json, so just created it manually.

### CODE REVIEW

**Comment by sarahbethfederman** [comment]
Oye. Why isn't the output tracked? I'm missing some checkout step but the bolt docs are unclear :(

File: config/customFormats.js [hunk 1]
```diff
@@ -13,6 +15,51 @@ const getLeaf = (object) => {
...
 };
 
+function MDX(dictionary) {
+  const topLevels = Object.entries(dictionary.properties);
+  let str = ``;
+
+  function MDXContent(dictionary, level, first) {
+    const category = dictionary[0];
+
+    let isToken = dictionary[1].value;
```
Comment (gwyneplaine) [inline config/customFormats.js]: This check will fail for cases where the value is set but falsey (0, "", null, undefined). We might want to do this instead: 
```suggestion
    let isToken = dictionary[1].hasOwnProperty("value");
```

File: config/customFormats.js [hunk 2]
```diff
@@ -13,6 +15,51 @@ const getLeaf = (object) => {
...
+
+    if (isToken) {
+      str += `\n| ${dictionary[0]} | ${dictionary[1].value} |`;
+    } else {
+      const subdictionary = Object.entries(dictionary[1]);
+      const hasProperties = subdictionary.every(function(item) {
+        return item[1].value;
```
Comment (gwyneplaine) [inline config/customFormats.js]: We'll probably want to do the same thing here 
```suggestion
        return item[1].hasOwnProperty("value");
```

**Review by gwyneplaine — CHANGES_REQUESTED** [review]
Two minor nits, otherwise very lgtm 🎉

File: config/customFormats.js (hunk 1 above)
Comment (sarahbethfederman) [reply config/customFormats.js]: Good catch, thanks!

**Review by gwyneplaine — APPROVED** [review]

File: config/customFormats.js (hunk 1 above)
Comment (gwyneplaine) [reply config/customFormats.js]: Sorry for the nit, was mostly just excited to use the "new" github suggestion feature 😂

**Comment by sarahbethfederman** [comment]
Haha don't apologise! Appreciate you pointing out the edge cases I missed,
it's really helpful

**Pull request merged**